providers/google/docs/changelog.rst

```diff
@@ -27,6 +27,16 @@
 Changelog
 ---------
 
+15.2.0
```

Comment:
I am not sure, but I think it should be `16.0.0`

Reply:
Changed to 16.0.0

Reply:
Removed deprecated links and split_tablename function